### ISSUE TITLE
Fix winmbr not working with two hard disks

### ISSUE THREAD

**Issue by adrian15** [rescatux/rescatux#9]
There is something wrong with winmbr using install-mbr command when dealing with several hard disks.

Either one has to do:
```
-e PARTITIONNUMBER
-p PARTITIONNUMBER
```
Or maybe we should try:
```
--enable =PARTITIONNUMBER
```
in order to fix it.

**Comment by adrian15** [comment]
Supposedly fixed on b02a840aa08a31b1f30058f99610fe202d5383b4 .
This needs to be confirmed.

Now at least the prompt will not be shown and the os will be inmediately booted.

**Comment by adrian15** [comment]
I think this has already been fixed in current 0.31 code.
